Tiltfile.sample

```diff
@@ -0,0 +1,127 @@
+# Welcome to Tilt!
```

Comment:
is sample used anywhere? should we keep it?

Reply:
I was keeping the sample as reference. I can see the case for and against it. I'll remove it for now.